src/parser/monk/mistweaver/modules/talents/SpiritOfTheCrane.js

```diff
@@ -132,30 +132,25 @@ class SpiritOfTheCrane extends Analyzer {
 
   statistic() {
     return (
-      <StatisticBox
+      <TalentStatisticBox
+        talent={SPELLS.SPIRIT_OF_THE_CRANE_TALENT.id}
         position={STATISTIC_ORDER.CORE(30)}
-        icon={<SpellIcon id={SPELLS.SPIRIT_OF_THE_CRANE_TALENT.id} />}
         value={`${formatNumber(this.manaReturnSotc)}`}
-        label={(
-          <dfn
-            data-tip={`
-              You gained a raw total of ${((this.manaReturnSotc + this.sotcWasted) / 1000).toFixed(0)}k mana from SotC with ${(this.sotcWasted / 1000).toFixed(0)}k wasted.<br>
-              You lost ${(this.totmOverCap + this.totmBuffWasted)} Teachings of the Monestery stacks
-            <ul>
-              ${this.totmOverCap > 0 ?
-              `<li>You overcapped Teachings ${(this.totmOverCap)} times</li>`
-              : ''
-              }
-              ${this.totmBuffWasted > 0 ?
-              `<li>You let Teachings drop off ${(this.totmBuffWasted)} times</li>`
-              : ''
-              }
-            </ul>
-            `}
-          >
-            Mana Returned
-          </dfn>
-        )}
+        label="Mana Returned"
+        tooltip={`
+          You gained a raw total of ${((this.manaReturnSotc + this.sotcWasted) / 1000).toFixed(0)}k mana from SotC with ${(this.sotcWasted / 1000).toFixed(0)}k wasted.<br>
+          You lost ${(this.totmOverCap + this.totmBuffWasted)} Teachings of the Monestery stacks
+          <ul>
+          ${this.totmOverCap > 0 ?
```

Comment:
For future reference the indentation here gets a bit weird, but I'll fix it when I merge this into `next`.